README.adoc

```diff
@@ -45,6 +45,10 @@ public static HoverflyRule hoverflyRule = HoverflyRule.inCaptureMode(classpath("
 // After the capturing, switch to inSimulationMode to spin up a stub server
 @ClassRule
 public static HoverflyRule hoverflyRule = HoverflyRule.inSimulationMode(classpath("simulation.json"));
+
+// Or you can use both approaches at once. If json file not present in capture mode, if present in simulation mode
+@ClassRule
+public static HoverflyRule hoverflyRule = HoverflyRule.inCaptureOrSimulationMode("simulation.json");
```

Comment:
I am really bad given names to things, so for me as simple as possible following the naming conventions of the project :)